src/rules/bn.toml

```diff
@@ -0,0 +1,52 @@
+
+# Replacements which are run before sentences are split
+replacements = [
+  ["ডাঃ","ডাক্তার"],
+  ["ডা.","ডাক্তার"],
+  ["ড.","ডাক্তার"],
+  ["নাং", "নম্বর"],
+  ["ন.", "নম্বর"],
+  ["নাং.", "নম্বর"],
+  ["নং", "নম্বর"],
+  ["মো.", "মোহাম্মদ"],
+  ["মোঃ", "মোহাম্মদ"]
+]
+
+min_trimmed_length = 3
+min_word_count = 2
+max_word_count = 20
+min_characters = 5
+may_end_with_colon = false
+quote_start_with_letter = true
+needs_punctuation_end = false
+needs_letter_start = true
+needs_uppercase_start = false
+broken_whitespace = ["  ", " ,", " .", " ?", " !", " ;"]
+
+allowed_symbols_regex = "[॰ঁংঃঅআইঈউঊঋঌএঐওঔকখগঘঙচছজঝঞটঠডঢণতথদধনপফবভমযরলশষসহ়ঽািীুূৃৄেৈোৌ্ৎৗয়ৠৡ৤০১২৩৪৫৬৭৮৯ৰৱ৳৹৺ਂ!\"',-\\./:;\\?`।„“‚‘’ ]"
+
+disallowed_symbols = [
+  '<', '>', '+', '*', '\', '#', '@', '^', '[', ']', '(', ')', '/',
+  'é', 'è', 'à', 'ç', 'Å',
+  'α', 'β', 'Γ', 'γ', 'Δ', 'δ', 'ε', 'ζ', 'η', 'Θ', 'θ', 'ι', 'κ',
+  'Λ', 'λ', 'μ', 'ν', 'Ξ', 'ξ', 'Π', 'π', 'ρ', 'Σ', 'σ', 'ς', 'τ',
+  'υ', 'Φ', 'φ', 'χ', 'Ψ', 'ψ', 'Ω', 'ω', 'a', 'b', 'c', 'd', 'e', 
+  'f', 'g', 'h', 'i', 'j', 'k', 'l', 'm', 'n', 'o', 'p', 'q', 'r', 
+  's', 't', 'u', 'v', 'w', 'x', 'y', 'z', 'A', 'B', 'C', 'D', 'E', 
+  'F', 'G', 'H', 'I', 'J', 'K', 'L', 'M', 'N', 'O', 'P', 'Q', 'R', 
+  'S', 'T', 'U', 'V', 'W', 'X', 'Y', 'Z'
+]
+
+even_symbols = ['"']
+
+matching_symbols = [
+  ["„", "“"],
+  ["(", ")"]
+]
+
+abbreviation_patterns = [
+  "([\u0995-\u09B9\u09CE\u09DC-\u09DF\u0985-\u0994\u09BE-\u09CC\u09D7\u09BC]+\\.+)+[\u0995-\u09B9\u09CE\u09DC-\u09DF\u0985-\u0994\u09BE-\u09CC\u09D7\u09BC]*"
```

Comment:
Could you add a comment to better explain what it does? I think that would be useful for whoever looks at it in the future. Example here: https://github.com/common-voice/cv-sentence-extractor/blob/main/src/rules/en.toml#L36

Reply:
Thanks for suggestion, have added some comments.